src/components/Button.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = ({ value }) => (
+  <div>{ value }</div>
+);
+
+Button.defaultProps = {
```

Comment:
I'm curious, I don't think you need this right?

![image](https://user-images.githubusercontent.com/31897434/77983162-2b334600-7306-11ea-9ad7-58b38c6ae556.png)

Reply:
I'm using a div as a button to avoid extra styling with css,  if you refer about defaultProps this is because eslint detect it as an error if props don't have a default value. 😄